zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Collection.java

```diff
@@ -202,6 +222,22 @@ public CollectionDescription create(HttpServletRequest request, HttpServletRespo
         return collection.getDescription();
     }
 
+    private void requireEditPermission(Session session) throws UnauthorizedException {
+        boolean canEdit = false;
+        try {
+            canEdit = permissionsService.canEdit(session);
+        } catch (IOException ex) {
+            throw new UnauthorizedException("You are not authorised to edit collections.");
+        }
+
+        if (!canEdit) {
+            warn().user(session.getEmail())
+                    .log("request unsuccessful user denied edit permission");
+
+            throw new UnauthorizedException("You are not authorised to edit collections.");
```

Comment:
Same here - remove the warning log and update the exception to include all the necessary details to understand the context of this error.